apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ScalarTypeAdapters.kt

```diff
@@ -0,0 +1,10 @@
+package com.apollographql.apollo.api
+
+expect class ScalarTypeAdapters(customAdapters: Map<ScalarType, CustomTypeAdapter<*>>) {
```

Comment:
What holds us to move this class to common?

Reply:
Answered here https://github.com/apollographql/apollo-android/pull/2062#discussion_r390797343